xcube/cli/gen2.py

```diff
@@ -52,6 +52,5 @@ def gen2(request_path: str,
     from xcube.cli._gen2.main import main
     main(request_path,
          output_path=output_path,
-         callback_api_url=callback_api_url,
```

Comment:
Maybe drop `callback_api_url` as CLI option entirely. Same for `output_path`. Just have the JSON requests will make this much easier to test and maintain. What do you think?